Shoko.Server/API/v3/Models/Shoko/WebUI.cs

```diff
@@ -217,6 +217,14 @@ public WebUISeriesFileSummary(SVR_AnimeSeries series, HashSet<EpisodeType> episo
                     if (file?.Media == null || location == null)
                         return null;
 
+                    return new { file, xref, location };
+                })
+                .Where(f => f != null);
+            var files = filesWithXrefAndLocation
```

Comment:
I've got no idea as to how expensive retreiving the SVR_VideoLocal & SVR_VideoLocal_Place records are, so I've gone down the 'safer' route of reusing these records.

If this is unnecessary, I'm more than willing to amend and re-request everything for the overview calculation rather than modifying this section.

Reply:
I didn't see this comment earlier, but yo answer; it's fine to do what you did… as long as you cache the result and don't run the query twice/thrice/etc.. If you append `.ToList()` after the last LINQ operation (the `.Where(…)` above in this case) then it will execute the query and turn it into a list which can more easily be reused for future LINQ query operations,